app/models/link_popularity.rb

```diff
@@ -1,11 +1,16 @@
+# frozen_string_literal: true
```

Comment:
Added to freeze strings, which will be the default in later Ruby versions:
https://rubocop.readthedocs.io/en/latest/cops_style/#stylefrozenstringliteralcomment